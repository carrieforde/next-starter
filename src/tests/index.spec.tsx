import React from "react";
import Home from "pages";
import { render } from "@testing-library/react";

describe("Home page", () => {
  it("should render the Home component", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
