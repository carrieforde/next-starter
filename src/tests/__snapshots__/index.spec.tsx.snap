// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render the Home component 1`] = `
<div>
  <h1>
    Welcome
  </h1>
</div>
`;
